_layouts/frontpage.html

```diff
@@ -136,227 +121,75 @@ <h3>{{scalaItem.shortTitle}}</h3>
         {% endif %}
       {% endfor %}
     </div>
-    <div class="call-to-action action-medium">
-      <a href="{{page.headerButtonUrl}}" class="button">{{page.headerButtonTitle}}</a>
-      <p class="align-bottom">or visit the <a href="/documentation/">Scala Documentation</a></p>
-    </div>
   </div>
 </section>
 
-<!-- IDEs for Scala -->
-<section class="ides">
+<!-- Scala backends -->
+<section class="runs">
   <div class="wrap">
-    <div class="heading-line">
-      <h2><span>IDEs for Scala</span></h2>
-    </div>
+    <h2>{{site.data.common.texts.scalaBackendsTitle}}</h2>
     <ul>
-      {% for ide in page.scalaIDEs %}
-      <li>
-        <a {% if ide.metals == true %}class="masterTooltip" title="{{site.data.common.texts.metalsSupportedInIDE}}"{% endif %} href="{{ide.url}}" target="_blank">
-          {% if ide.metals == true %}
-            <span class="bullet">
-              <img src="/resources/img/frontpage/scalameta-logo.png" alt="">
-            </span>
-          {% endif %}
-          <img src="{{ide.icon}}" alt="{{ide.name}}">
-          <span>{{ide.name}}</span>
-        </a>
+      {% for backend in page.scalaBackends %}
+      <li class="masterTooltip" title="{{backend.description}}">
+        <span>
+          <a href="{{backend.link}}"><img src="{{backend.icon}}" alt="{{backend.description}}"></a>
+        </span>
+        {% if backend.beta == 1 %}<span class="beta">beta</span>{% endif %}
       </li>
-        {% unless forloop.last %}
-          <li></li>
-        {% endunless %}
+      {% unless forloop.last %}<li></li>{% endunless %}
       {% endfor %}
     </ul>
   </div>
 </section>
 
-<!-- Run Scala in your browser -->
-{% include scastie.html %}
```

Comment:
I think it was nice to have this one. At least, the ability to have executable code snippets from the front page is cool. Maybe the “Scala in a Nutshell” examples could be made executable? (that would probably be a big effort though)

Reply:
Note that there is now a "Playground" link all the way at the top of page.

I think it's only worth spending the space on this on the front page if we actually give people sample snippets they can run, as you suggest. So I'd be in favor of bringing this back in that form in the future. At present, it just dumps visitors into a thing that isn't useful unless you already know Scala, so I think it should go.